src/server/routes/controllers/email.js

```diff
@@ -0,0 +1,119 @@
+const responseManager = require('./utilities/responseManager')
+const config = require('../../config.json')
+const nodemailer = require('nodemailer')
+const logger = require('../../logger')
+const crypto = require('crypto')
+const _ = require('lodash')
+
+const testAccount = {
+    email: process.env.EMAIL || config.email.email,
+    password: process.env.EMAIL_PASSWORD || config.email.password,
+    host: process.env.EMAIL_HOST || config.email.host
+}
+
+const SMTP_OPTIONS = { //TODO: Add SMTPS
```

Comment:
Do we plan it in this PR? If not, please create a ticket for SMTPS instead, and remove the comment.